imod_coupler/drivers/dfm_metamod/mf6_wrapper.py

```diff
@@ -30,11 +30,76 @@ def set_river_stages(
         ValueError
             the size of the provided stage array does not match the expected size
         """
+        stage = self.get_river_stages(mf6_flowmodel_key, mf6_river_pkg_key)
+        if new_river_stages is None or len(new_river_stages) != len(stage):
+            raise ValueError(f"Expected size of new_river_stages is {len(stage)}")
+        stage[:] = new_river_stages[:]
+
+    def get_river_stages(
```

Comment:
do you need a full function for this? is it not cleaner to just get the stage in the previous function? we don't need the mf-stage anywhere else right?

Reply:
true, but the setter needs to do a "get" first because it needs to multiply  the river stages with the mask.
I split the "get"and the "set" because it provides a better separation of concerns, and having a "get" makes the mf6_wrapper  more reusable.